app/mailers/person_mailer.rb

```diff
@@ -256,10 +256,13 @@ def new_member_notification(new_member, community, admin)
     @person = new_member
     @email = new_member.emails.last.address
     with_locale(admin.locale, community.locales.map(&:to_sym), community.id) do
-      premailer_mail(:to => admin.confirmed_notification_emails_to,
-                     :from => community_specific_sender(community),
-                     :subject => "New member in #{@community.full_name(@person.locale)}",
-                     :template_name => "new_member_notification")
+      address = admin.confirmed_notification_email_to
+      if !address.blank?
```

Comment:
The premailer_mail call below does not actually send the email. Instead this returns an email to be sent to at the calling side (the job) you should also handle now this method returning nil. I'm not sure that it's the case today.